connectivity/check/policy.go

```diff
@@ -24,43 +24,12 @@ import (
 	"github.com/cilium/cilium-cli/k8s"
 )
 
-type ExitCode int16
```

Comment:
Consider moving this refactor in a separate commit to make it easier on reviewers